crates/typst-pdf/src/color.rs

```diff
@@ -65,18 +66,18 @@ impl ColorSpaces {
     /// PDF file.
     pub fn write_functions(&self, chunk: &mut Chunk, refs: &ColorFunctionRefs) {
         // Write the sRGB color space.
-        if self.use_srgb {
+        if let Some(id) = refs.srgb {
```

Comment:
@elegaanz does this change look ok?

Reply:
Yes, I don't think it should change the final result (if the point was just to avoid `unwrap`ing).